addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -347,10 +348,13 @@ def _l10n_es_edi_get_invoices_info(self, invoices):
                         invoice_node.setdefault('TipoDesglose', {})
                         invoice_node['TipoDesglose'].setdefault('DesgloseTipoOperacion', {})
                         invoice_node['TipoDesglose']['DesgloseTipoOperacion']['PrestacionServicios'] = tax_details_info_service_vals['tax_details_info']
-                    if tax_details_info_consu_vals['tax_details_info']:
+                    elif tax_details_info_consu_vals['tax_details_info']:
                         invoice_node.setdefault('TipoDesglose', {})
                         invoice_node['TipoDesglose'].setdefault('DesgloseTipoOperacion', {})
                         invoice_node['TipoDesglose']['DesgloseTipoOperacion']['Entrega'] = tax_details_info_consu_vals['tax_details_info']
+                    else:
```

Comment:
Instead of the `else`, we could maybe do something like `if invoice_node.get('TipoDesglose')` ?

Reply:
I'll adapt accordingly this too.